lib/fastlane/actions/xcodebuild.rb

```diff
@@ -112,8 +120,13 @@ def self.run(params)
           if hash_args = hash_to_args(params)
             xcodebuild_args += hash_args
           end
+
+          buildlog_path ||= params[:buildlog_path]
         end
 
+        # By default we put xcodebuild.log in the current folder
+        buildlog_path ||= "."
```

Comment:
Thanks for reviewing ! 
My experience with various cli tools is that usually they create log files in the folder I run them.
I will modify that behaviour. 

As a note, creating all logs in ```~/Library/Logs/xctest ``` looks reasonable when there is one user running one ```fastlane``` at a time. But on a CI box where there are running several jobs at a time, that is not a solution hence the need to have it in the workspace of the job.